generators/app/prompts.js

```diff
@@ -104,5 +104,21 @@ module.exports = [
     message: 'Do you want lazy loading?',
     default: false,
     when: props => props.ui !== 'ionic'
+  },
```

Comment:
A `--hash-strategy` or `--location-strategy` option would be preferable here instead of an additional question as it's not a common use case, and using hash should be avoided whenever possible according to the Angular team.

We try to maintain the number of questions as low as possible and move other settings to options to keep the generation process as focused as possible. 😉

Reply:
The hash strategy is more suitable for the mobile application scenarios.

OK, agree absolutely. `--location-strategy`